### PR TITLE
Added new render function to create batches of imagse from multiple p…

### PR DESCRIPTION
**Pull request by cacadam89**
Added batch image rendering function in render class.

Modified main to call this function to generate test images during training when args are set.

Updated teddy_bear config to have appropriate args to test these changes

***Important*** I also changed the requires_grad to be True in the utils load checkpoint function. This means the model can continue to be trained (but will be larger). We should figure out what behavior we actually want going forward, and consider making this controlled by the arguments.

### CODE REVIEW

File: render_utils.py [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
 import nerf
 import numpy as np
+import os
 import torch
 import utils
 
 import torch.nn.functional as F
 
+#Converts an image's pixels values from [0, 1] to [0, 255]
+to8b = lambda x : (255*np.clip(x,0,1)).astype(np.uint8)
```
Comment (pculbertson) [inline render_utils.py]: nitpick: isn't this already defined in `utils`? you might just want to use `utils.to8b` or just define it here

File: utils.py [hunk 2]
```diff
@@ -200,12 +200,14 @@ def load_checkpoint(coarse_model, fine_model, optimizer, args):
         # Load model
         coarse_model.load_state_dict(ckpt['coarse_model_state_dict'], strict=False)
         for param in coarse_model.parameters():
-            param.requires_grad = False
+            param.requires_grad = True
+            # param.requires_grad = False
```
Comment (pculbertson) [inline utils.py]: I would just delete this line rather than leave it commented -- or maybe pass `requires_grad` as an argument to the load function?

**Review by pculbertson — APPROVED** [review]
Looks great to me, just nitpicked a couple stylistic things!

File: render_utils.py (hunk 1 above)
Comment (cacadam89) [reply render_utils.py]: lol, this was just me not knowing i could access a lambda function from another module... that is def the right call

File: utils.py (hunk 2 above)
Comment (cacadam89) [reply utils.py]: Agreed. See latest push for this change (and the to8b comment above)

**Comment by cacadam89** [comment]
I am calling this good for now

**Pull request merged**